pulp_smash/config.py

```diff
@@ -135,11 +136,13 @@ def __init__(  # pylint:disable=too-many-arguments
             auth=None,
             verify=None,
             version=None,
+            pulp_database=None,
             cli_transport=None):
         """Initialize this object with needed instance attributes."""
         self.base_url = base_url
         self.auth = auth
         self.verify = verify
+        self.pulp_database = pulp_database
```

Comment:
If the user doesn't provide a value, this attribute will be `None`, which seems wrong.